docs/reference/settings/security-settings.asciidoc

```diff
@@ -1633,6 +1633,23 @@ at the OpenID Connect Provider.
 The OAuth 2.0 Client Secret that was assigned to {es} during registration
 at the OpenID Connect Provider.
 
+// tag::rp-client-auth-method-tag[]
+`rp.client_auth_method` {ess-icon}::
+(<<static-cluster-setting, Static>>)
+The Client authentication method used by elasticsearch as a client to authenticate
+to the OpenID Connect Provider. Can be `client_secret_basic`, `client_secret_post`,
+or `client_secret_jwt`. Defaults to `client_secret_basic`.
+// end::rp-client-auth-method-tag[]
+
+// tag::rp-client-auth-jwt-signature-algorithm[]
+`rp.client_auth_signature_algorithm` {ess-icon}::
+(<<static-cluster-setting, Static>>)
+The signature algorithm that {es} uses to sign the JWT with which it authenticates
+as a Client to the OpenID Connect Provider when `client_secret_jwt` is selected for
```

Comment:
Unless this is a specific client that is always capitalized, I think it should be lowercase:
```suggestion
as a client to the OpenID Connect Provider when `client_secret_jwt` is selected for
```